src/pages/resultsView/download/DownloadUtils.ts

```diff
@@ -371,6 +372,8 @@ export function generateCaseAlterationData(
                 // that's why we need a map here
                 caseAlterationData[key].oqlData[data.oql.oql_line] = generateOqlData(datum, geneAlterationDataByGene, molecularProfileIdToMolecularProfile);
                 updateOqlData(datum, caseAlterationData[key].oqlData[data.oql.oql_line], molecularProfileIdToMolecularProfile);
+                caseAlterationData[key].oqlDataByGene[data.oql.gene] = caseAlterationData[key].oqlDataByGene[data.oql.gene] !== undefined ? _.merge(generateOqlData(datum, geneAlterationDataByGene, molecularProfileIdToMolecularProfile), caseAlterationData[key].oqlDataByGene[data.oql.gene]) : generateOqlData(datum, geneAlterationDataByGene, molecularProfileIdToMolecularProfile);
```

Comment:
this code here is too dense.  very hard to read.   i would avoid using ternary when there's so much going on.  if statement would be clearer.  or you could spread ternary across several lines with linebreaks

Reply:
Thank Aaron, I will update and clean the code then let you know!